docs/en/guides/developer/working-with-json/json-load-data.md

```diff
@@ -0,0 +1,129 @@
+---
+slug: /en/guides/developer/working-with-json/json-load-data
+sidebar_label: Loading JSON
+sidebar_position: 1
+description: Loading JSON into ClickHouse
+---
+
+# Loading JSON in 5 steps
+
+This guide walks through the process to load logging data that is 
+in a JSON formatted file in S3.  In order to do this:
+- Examine the file format by selecting one row using the S3 function
+- Create a table to store the data in ClickHouse
+- Load a single row of nested JSON
+- Verify the correct storage of the nested JSON
+- Import the dataset from S3
+
+## Examine the structure of the JSON file
```

Comment:
Let's maybe also add `DESCRIBE` query? And also worth mentioning that we have not only JSONEachRow format, but also some other JSON formats that can fit users data

Reply:
yes, DESCRIBE is needed.

For the other formats, I will add a section in Notes at the bottom.  For beginner tutorials like this, I want to give them a step-by-step guide that just works without a lot of discussion about other options or links to other information until the end, and then at the end, provide more information.  I will look at the input / output formats for more info.

Reply:
added describe